Change gbak output of errors and warnings to stderr instead of stdout

XaBbl4 opened the pull request:
Currently in standalone application mode when redirect to the standard stream, for example:
`gbak ... > /path/to/stdout.log`
some error and warning messages may be missed, which may cause inconvenience.

For example, during when restore we may see an error message:
```
gbak: ERROR:Database is not online due to failure to activate one or more indices.
gbak: ERROR:    Run gfix -online to bring database online without active indices.
```
but in order to find out which index remains inactive, we need to look at the entire log, and in it we can already find:
```
gbak:cannot commit index TEST_INDEX
gbak: ERROR:attempt to store duplicate value (visible to active transactions) in unique index "TEST_INDEX"
gbak: ERROR:    Problematic key value is ("ID" = 1)
```

Although in the global community it is accepted to output all errors and warnings to stderr.

This patch fixes this bug. I tried to cover all the cases found, but may have missed something.
I believe `BURP_print_status` should always output to stderr, regardless of whether the `err` argument is set to true or not. The `err` argument is now only responsible for setting status in service mode. Also `BURP_print_warning` should always output to stderr.

XaBbl4 committed:
Fix output errors and warnings to stderr in gbak utility

AlexPeshkoff reviewed (changes requested):
Andrey, may be I was not enough explicit. I suggest to:
1. Complete cleanup process in restore.epp. There is no need to reproduce all bugs - just find all BURP_print calls with false parameter. Regarding previous comments about about errors 171 & 343 - it's completely OK to sent output to stderr for both of them. Like all other invocations of BURP_error.

2. After cleanup finished - may be argument `err` should be removed from BURP_print() at all? For a few remaining calls in burp.cpp like
`BURP_print(false, 91, FB_VERSION);`
may be invite new function name?

XaBbl4 committed:
Remove err argument from BURP_print* functions

All messages from these functions are now printed to standard error stream.
BURP_print with output version has been replaced by BURP_message (same result).
Calls BURP_error_redirect with NULL status argument replaces BURP_error with abort argument, because calling BURP_print_status only adds call overhead.

XaBbl4 committed:
Remove the serviceFlag for "error accessing blob field" in get_array
